qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: Send CV
+
+  Scenario: positive sending CV
```

Comment:
For each scenario, you need to specify its number. For example, 1.2.1_01 - the first one, 1.2.1_02 - the second one etc.

Reply:
done